acme/acme/client.py

```diff
@@ -577,10 +581,15 @@ def new_account(self, new_account):
 
         :param .NewRegistration new_account:
 
+        :raises .ConflictError in case the account already exist
```

Comment:
This should probably be.
```
:raises .ConflictError: in case the account already exists
```
You need the colon after `ConflictError` for the links in our docs to build properly (see https://certbot.eff.org/docs/contributing.html#updating-the-documentation if you want to build our docs yourself) and I think "exists" rather than "exist" is more grammatically correct.